main.go

```diff
@@ -445,6 +480,122 @@ func deleteRun(_ *cobra.Command, args []string) {
 	fmt.Fprintf(stdout, "Successfully deleted secret if it existed: %s\n", object)
 }
 
+func isEditorAvailable(e string) (bool, error) {
+	cmd := exec.Command("which", "-s", e)
+	if err := cmd.Start(); err != nil {
+		return false, err
+	}
+	if err := cmd.Wait(); err != nil {
+		if _, ok := err.(*exec.ExitError); ok {
+			return false, nil
+		}
+		return false, err
+	}
+	return true, nil
+}
+
+func findEditor() error {
+	if editor == "" {
+		editor = os.Getenv("VISUAL")
+	}
+	if editor == "" {
+		editor = os.Getenv("EDITOR")
+	}
+	if editor != "" {
+		return nil
+	}
+
+	editorsToTry := []string{"vi", "emacs", "nano", "pico"}
+	for _, e := range editorsToTry {
+		if available, err := isEditorAvailable(e); err != nil {
+			return err
+		} else if available {
+			editor = e
+			return nil
+		}
+	}
+	return errors.New("unable to determine editor. manually specify using `--editor` flag and try again")
+}
+
+func editRun(_ *cobra.Command, args []string) {
+	bucket, object, err := parseRef(args[0])
+	if err != nil {
+		handleError(err, 2)
+	}
+
+	ctx := cliCtx()
+	client, err := berglas.New(ctx)
+	if err != nil {
+		handleError(err, 2)
```

Comment:
For all these errors, we should wrap "the secret was not updated" for clarity.

```go
func notUpdated(err error) error {
  return errors.Wrap(err, "the secret was not updated")
}

handleError(notUpdated(err), 1)
```

Just to make it clear to the user

Reply:
Added the wrap to this particular error - however other errors seem clear on their own?